aiohttp/web_protocol.py

```diff
@@ -105,7 +104,7 @@ async def log(
         self.access_logger.log(request, response, self._loop.time() - request_start)
 
 
-@attr.s(auto_attribs=True, frozen=True, slots=True)
+@dataclasses.dataclass(frozen =True)
```

Comment:
Having a whitespace before `=` is a style violation. The linter would make the CI red.
```suggestion
@dataclasses.dataclass(frozen=True)
```